config/initializers/secure_headers.rb

```diff
@@ -13,9 +13,13 @@
     :enforce     => true,
     :default_src => 'self',
     :frame_src   => 'self',
-    :connect_src => 'self ws: wss:',
+    :connect_src => %w(self ws: wss:),
     :style_src   => 'inline self',
-    :script_src  => 'eval inline self',
-    :img_src     => ['self', '*.gravatar.com']
+    :script_src  => %w(eval inline self),
+    :img_src     => %w(self *.gravatar.com)
   }
+  if Rails.env.development?
```

Comment:
nitpick, add a comment to mention that's for Webpack, it's not obvious if it's the first time you encounter it